inference-engine/src/legacy_api/src/cnn_network_int8_normalizer.cpp

```diff
@@ -1003,12 +1003,14 @@ void CNNNetworkInt8Normalizer::QuantizeConvolutionOrFullyConnected(CNNLayer::Ptr
                 symQuant = *(intervals.begin());
             }
             std::set<double> divs;
-            prev = 0.f;
-            for (auto it = individualsG.begin(); it != individualsG.end(); it++) {
-                if (prev) {
-                    divs.insert((*it - prev) / symQuant);
+            if (symQuant != 0.) {
+                prev = 0.f;
+                for (auto it = individualsG.begin(); it != individualsG.end(); it++) {
+                    if (prev) {
+                        divs.insert((*it - prev) / symQuant);
+                    }
+                    prev = *it;
```

Comment:
Nitpick: for-each loop is applicable here?

```cpp
for (const auto& individualG : individualsG) {
    // the same body except renaming it to individualG
}
```

Reply:
Here is zero division protection.

Reply:
Sure, but my question is not about loop body, but about `for` statement itself. I don't see a reason so far to deal with iterators here.